integration/testrunner/grpcrunner/grpc.go

```diff
@@ -320,7 +344,7 @@ func createConnection(tb testing.TB, addr string, idToken string) *grpc.ClientCo
 	addr = addr + ":443"
 	conn, err := grpc.Dial(addr, grpc.WithPerRPCCredentials(rpcCreds), grpc.WithTransportCredentials(creds))
```

Comment:
Can be a followup, but we should be using DialContext instead: https://pkg.go.dev/google.golang.org/grpc#DialContext

Reply:
Done